README.md

```diff
@@ -1,37 +1,80 @@
-# ui
-This provides a graphical interface to run and build your GPTScripts.
+# Acorn Desktop
 
-![Chat Image](assets/chat.png)
+💫 **The best way to build, use, and share AI Assistants.** 💫 
 
-## Running
-The UI is baked directly into the [gptscript](https://docs.gptscript.ai/) CLI. To run the UI, simply run the following command:
+https://github.com/user-attachments/assets/b9cfa1c3-ba76-4193-84bf-115096b1a3ee
```

Comment:
What is this link for?

Reply:
video recording

Reply:
Ah, it doesn't work in the mobile app for some reason, but I see it on desktop